djangocms_alias/cms_toolbars.py

```diff
@@ -62,9 +61,12 @@ def add_aliases_link_to_admin_menu(self):
         if not self.request.user.has_perm('djangocms_alias.change_category'):
             return
         admin_menu = self.toolbar.get_or_create_menu(ADMIN_MENU_IDENTIFIER)
-        admin_menu.add_link_item(
-            _('Aliases'),
-            url=admin_reverse(CATEGORY_LIST_URL_NAME),
+        url = reverse("admin:{}_aliascontent_changelist".format(
+            AliasContent._meta.app_label)
```

Comment:
This looks a bit odd with the bracket at the end, could you add it to a new line?

Reply:
Done